README.md

```diff
@@ -6,6 +6,8 @@
 
 RuboCop helps you enforce many of the guidelines outlined in the community [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide). Most aspects of its behavior can be tweaked via various [configuration options](https://github.com/bbatsov/rubocop/blob/master/config/default.yml), which are set in a **.rubocop.yml** file.
 
+You can find some basic setup instructions and links to the RuboCop OSS project below. For additional Code Climate-specific config details, check out our [RuboCop engine documentation here](https://docs.codeclimate.com/docs/rubocop).
```

Comment:
Personal opinion, but I think "here" is redundant in the link text. It's a link, and it's "RuboCop engine documentation": I don't think more is required.